Dangerfile

```diff
@@ -13,6 +13,13 @@ if en_docs_modified ^ cn_docs_modified
   warn("Consider **also** updating the #{ en_docs_modified ? "English" : "Chinese" } docs. For Chinese translations, request the modified file(s) to be added to the list [here](https://github.com/Moya/Moya/issues/1357) for someone else to translate, if you can't do so yourself.")
 end
 
+# Warn, asking to update Chinese docs if only English README are updated and vice-versa
+en_readme_modified = git.modified_files.grep(%r{Readme.md}).empty?
+cn_readme_modified = git.modified_files.grep(%r{Readme_CN.md}).empty?
+if en_readme_modified ^ cn_readme_modified
+  warn("Consider **also** updating the #{ en_readme_modified ? "English" : "Chinese" } README. For Chinese translations, request the modified file(s) to be added to the list [here](https://github.com/Moya/Moya/issues/1357) for someone else to translate, if you can't do so yourself.")
```

Comment:
Same thing here on `en_readme_modified ? "English" : "Chinese" `: if only English Readme was modified, you should ask for the Chinese change as well (so basically make it `"Chinese" : "English"`).